src/frontend/shared/components/FriendsPanel.tsx

```diff
@@ -22,10 +23,10 @@ class FriendList extends React.Component<FriendListProps, {}> {
         return (
             <div>
                 <a className="glyphicon glyphicon-user right-open-button" onClick={() => this.toggleFriendList()}/>
-                <div ref={this.right_collapse} className="right_collapse">
+                <div ref={this.rightCollapse} className="right_collapse">
                 <ul className="myFriend">
```

Comment:
Do you think it is usefult to extract the friend list as a separate component? Since side panel is not logically linked to Friend list.

Reply:
Hmm... That's a good point. I think if we ever need to reuse the Friends List in another place, we can extract it out. But currently I don't think it's reusable so maybe another PR for that in the future if needed.